crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,66 @@
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::Query,
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+#[derive(Default, Component)]
+pub struct HasSceneInstance(Option<InstanceId>);
```

Comment:
In the code below, this field is referenced repeatedly as .0 and it's hard to follow. Can we change this to a named struct, with an `id` field?

Reply:
implemented `Deref` and `DerefMut` for it